spec/features/api_particulier/authorization_request_spec.rb

```diff
@@ -239,6 +239,10 @@
               expect(page).to have_css('#contact_technique')
               expect(page).to have_css('#contact_technique_its_me')
             end
+
+            it 'displays old tokens accordion' do
+              expect(page).to have_css('#old_tokens_accordion_button')
+            end
           end
```

Comment:
Et du coup ici je vérifierai qu'il n'y a PAS le(s) jeton(s) actif(s) (not sure pour le `(s)` ?)

Reply:
Théoriquement, il est possible d'avoir plusieurs jetons actifs (pas de limite technique). Pratiquement et vu les mécanismes not possible